math.md

```diff
@@ -95,20 +95,18 @@ focus on testing something that creates them.
 So my first test looks like this:
 
 ```go
-package clockface_test
+package clockface
```

Comment:
This is an example of a test file that seemed to think it was testing functionality in a different package. I changed the package name and removed "clockface" in front of the functions.